boards/v2/qemu/qemu_arc/Kconfig.qemu_arc

```
@@ -0,0 +1,26 @@
+# Copyright (c) 2020 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_QEMU_ARC
+	bool
```

bool not needed.

And as all board identifier are doing: `select BOARD_QEMU_ARC`, then let's move that up, and remove redundant code.
```suggestion
	select BOARD_QEMU_ARC
```

---

Fixed